cli/src/cli-cmd.c

```diff
@@ -379,14 +379,12 @@ cli_cmd_submit(struct rpc_clnt *rpc, void *req, call_frame_t *frame,
 }
 
 int
-cli_cmd_pattern_cmp(void *a, void *b)
+cli_cmd_pattern_cmp(const void *a, const void *b)
 {
-    struct cli_cmd *ia = NULL;
-    struct cli_cmd *ib = NULL;
+    struct cli_cmd *ia = (struct cli_cmd *)a;
+    struct cli_cmd *ib = (struct cli_cmd *)b;
     int ret = 0;
 
-    ia = a;
-    ib = b;
     if (strcmp(ia->pattern, ib->pattern) > 0)
```

Comment:
You could just return the result of `strcmp()`. A negative or positive number is enough. It's not required to return -1 or 1.

Reply:
Sure.